src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -12,12 +12,12 @@
     <PackageReference Include="Obsolete.Fody" Version="5.2.0" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.1" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="0.6.2" PrivateAssets="All" />
-    <PackageReference Include="RabbitMQ.Client" Version="[5.1.0, 6.0.0)" />
+    <PackageReference Include="RabbitMQ.Client" Version="[5.1.2, 6.0.0)" />
   </ItemGroup>
 
   <PropertyGroup>
     <PackageId>NServiceBus.RabbitMQ</PackageId>
     <Description>RabbitMQ support for NServiceBus</Description>
   </PropertyGroup>
 
-</Project>
+</Project>
```

Comment:
nitpick: Let's add the newline back.

Reply:
What's the purpose of the blank new line?

Reply:
Its the standard way that project files are defined in the template, so let's keep it consistent.

Reply:
> Its the standard way that project files are defined in the template, so let's keep it consistent.

Could you please point me to the standard to see it? I am not aware of this requirement. While it's a nitpick, if this is something we're going to apply to all the projects, I'd rather do it now, understanding the rationale.

Reply:
Discussed and it's not needed.

Reply:
I can see that it's already inconsistent in the repo, so I guess it doesn't matter.